cypress/e2e/models/migration/applicationinventory/analysis.ts

```diff
@@ -288,6 +290,38 @@ export class Analysis extends Application {
         }
     }
 
+    public static analyzeAll(params: Analysis): void {
+        Analysis.open();
+        selectCheckBox("#bulk-selected-items-checkbox");
+        cy.contains(button, analyzeButton).should("be.enabled").click();
+        params.selectSourceofAnalysis(params.source);
+        params.isNextEnabled();
+        cy.contains(button, next).click();
+        params.selectTarget(params.target);
+        cy.contains(button, next).click();
+        params.scopeSelect();
+        if (params.customRule) {
+            params.uploadCustomRule();
+        }
+        if (params.customRuleRepository) {
+            params.fetchCustomRules();
+        }
+        cy.contains(button, next).click();
+        if (params.excludeRuleTags) {
+            params.tagsToExclude();
+        }
+        if (params.enableTransaction) {
+            params.enableTransactionAnalysis();
+        }
+        if (params.disableTagging) {
+            params.disableAutomatedTagging();
+        }
```

Comment:
We can call the analyze  method here after selecting the bulk check box

Reply:
We can't because the method is not static, it is called on `this` so it would be a weird notation if we call it as an instance method. But I can do something similar to reuse code